components/camel-rabbitmq/src/main/java/org/apache/camel/component/rabbitmq/RabbitMQConsumer.java

```diff
@@ -115,16 +112,13 @@ private void startConsumer(Channel channel) throws IOException {
     protected void doStart() throws Exception {
         executor = endpoint.createExecutor();
         log.debug("Using executor {}", executor);
-        try {
-            openConnection();
-            startConsumers();
-        } catch (Exception e) {
-            log.info("Connection failed, will start background thread to retry!", e);
-            reconnect();
-        }
+        startConsumers();
     }
 
-    private void reconnect() {
+    private synchronized void reconnect() {
+        if (startConsumerCallable != null) {
```

Comment:
We dont want all the consumers to disconnect at the same and all start a new thread.